certificateshop-logic/src/main/java/com/epam/esm/dao/impl/jdbc/ColumnNames.java

```diff
@@ -0,0 +1,99 @@
+package com.epam.esm.dao.impl.jdbc;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Stores names of the database tables columns, which are used by {@link GiftCertificateExtractor}
+ * {@link CertificateTagMapper} and {@link GiftCertificateMapper} to fetch data from a ResultSet.
+ */
+public class ColumnNames {
```

Comment:
Util classes which have only static methods should be final as well because there are no methods to override

Reply:
Done